web/models.py

```
@@ -2787,3 +2787,105 @@ class ScheduledPost(models.Model):
+
+    # Order for sequential display
+    order = models.PositiveIntegerField(default=0)
+
+    class Meta:
+        ordering = ["order"]
+
+    def __str__(self):
+        return self.title
+
+    @property
+    def progress_percentage(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `progress_percentage`

(ANN201)